modules/peers.js

```diff
@@ -90,6 +90,29 @@ __private.countByFilter = function(filter, cb) {
 	);
 };
 
+/**
+ * Shuffles peers (using Fisher-Yates-Durstenfeld shuffle algorithm).
+ *
+ * @todo Add @param tags
+ * @todo Add @returns tag
+ * @todo Add description of the function
+ */
+__private.shuffle = function(array) {
+	let m = array.length;
+	let t;
+	let i;
+	// While there remain elements to shuffle
+	while (m) {
+		// Pick a remaining element
+		i = Math.floor(Math.random() * m--);
+		// And swap it with the current element
+		t = array[m];
+		array[m] = array[i];
+		array[i] = t;
+	}
+	return array;
+};
```

Comment:
The ES6 equivalent can be
```
function shuffle(a) {
    for (let i = a.length - 1; i > 0; i--) {
        const j = Math.floor(Math.random() * (i + 1));
        [a[i], a[j]] = [a[j], a[i]];
    }
    return a;
}
```

We are using `lodash` in this module and it contains `shuffle` function with same Fisher-Yates algorithm, then why we need to create our new function.

Reply:
I thought about this too!
since we are using lodash, we should've probably just use shuffle.
However, I think the intention was not to change the original code.

Reply:
@nazarhussain @shuse2 Yes, this function was already there (I just moved it) and I didn't want to use a different algorithm and risk breaking something which works. This is a change we can do later.